plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -61,6 +61,7 @@ type CloudWatch struct {
 	MaxValuesPerDatum  int                      `toml:"max_values_per_datum"`
 	MetricConfigs      []MetricDecorationConfig `toml:"metric_decoration"`
 	RollupDimensions   [][]string               `toml:"rollup_dimensions"`
+	DropOriginConfigs  map[string][]string      `toml:"drop_original_metrics"`
```

Comment:
Nit: the attribute is drop origin config, but the TOML maps to `drop_original_metrics`